types/jshint/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for jshint 2.12
+// Project: http://jshint.com/
+// Definitions by: Amin Yahyaabadi <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+export interface LintError {
+    id: string;
+    raw: string;
+    code: string;
+    evidence: string;
+    line: number;
+    character: number;
+    scope: string;
+    reason: string;
+    [key: string]: any;
```

Comment:
Isn't that limited to nullable `a`,`b`,`c`,`d` only?
https://github.com/jshint/jshint/blob/master/src/jshint.js#L407-L419

Reply:
I thought this a, b, c, d is arbitrary! But it seems you are right
```suggestion
  a?: any
  b?: any
  c?: any
  d?: any
```